content/docs/command-reference/remote/modify.md

```diff
@@ -58,6 +58,15 @@ manual editing could be used to change the configuration.
 
 The following config options are available for all remote types:
 
+- `url` - the remote location can always be modified. This is how DVC determines
+  what type of remote it is, and thus which other config options can be modified
+  (see each type in the next section for more details). Here's a _local remote_
+  example:
+
+  ```dvc
+  $ dvc remote modify myremote url /home/user/dvcstore
```

Comment:
let's do s3 example and can even explain about s3 prefix, or do another example if needed in a separate block?

Reply:
OK added S3 example before local example although I'm not sure it's needed as it's also in the S3 section. The S3 URL format is also explained in its section. See 3b7898e3